TempSensor.ino

```diff
@@ -144,15 +144,97 @@ uint16_t MLX90614::read16(uint8_t a) {
 
 MLX90614 mlx = MLX90614();
 
+// lcd constants
+#define LCD_BRIGHTNESS_HIGH 1
+#define LCD_BRIGHTNESS_LOW 0
+
+// lcd pins
+#define LCD_RS_PIN 10
+#define LCD_EN_PIN 9
+#define LCD_D4_PIN 7
+#define LCD_D5_PIN 6
+#define LCD_D6_PIN 5
+#define LCD_D7_PIN 4
+#define LCD_BL_PIN 3
+#define LCD_BR_PIN 11
+
+// trigger pins
+#define TRIGGER_PIN 2
+
+// laser pins
+#define LASER_PIN 8
+
+// operational settings
+#define REFRESH_TEMP_READING 200
+
 // feature flags
-const int flagMovingAverageEnable = false;
+#define FLAG_MOVING_AVG_ENABLE false
 
-const int refreshTempReading = 500;
+class LiquidCrystalBacklight{
+  private:
+    uint8_t _backlight_pin;
+    uint8_t _brightness_pin;
+  public:
+    LiquidCrystalBacklight(uint8_t bl, uint8_t br)
+    {
+      _backlight_pin = bl;
+      _brightness_pin = br;
+    }
+
+    void display(){
+      digitalWrite(_backlight_pin, HIGH);
+      analogWrite(_brightness_pin, LCD_BRIGHTNESS_HIGH);
+    }
+    void noDisplay(){
+      digitalWrite(_backlight_pin, LOW);
+      analogWrite(_brightness_pin, LCD_BRIGHTNESS_LOW);
+    }
+};
+
+class Trigger {
+  public:
+    void begin(){
+      pinMode(TRIGGER_PIN, INPUT);
+      digitalWrite(TRIGGER_PIN, HIGH);
```

Comment:
We are not able to set this to 'LOW' as the simulated circuit doesn't start up.

Reply:
Not sure what you mean by simulated circuit does not start up. The code looks fine to me.

Reply:
Got it now